wormhole-connect/src/sdk/gasEstimates.ts

```diff
@@ -195,6 +199,11 @@ const getGasFeeFallback = async (
     }
   }
 
+  // TODO: look into cosmosdk/sei tx fees simulation/estimation
+  if (fromChainId === MAINNET_CHAINS.sei) {
+    return '0';
```

Comment:
TODO

Reply:
Done. Had to do it through the aggregator sdk since the cosmos client requires a wallet as a parameter, even if it's for tx simulation/gas simulation.